src/css/variables.scss

```diff
@@ -0,0 +1,79 @@
+// Fonts
+$base-font: 'Source Code Pro', monospace;
+$logo-font: 'Varela', sans-serif;
+
+// Font-sizes
+$logo-text: 3rem;
+$big-text: 1.5rem;
+$medium-text: 1.25rem;
+$small-text: 1rem;
+$tiny-text: 0.75rem;
+
+// Colors
+$primary-bg-color: hsla(206, 100, 97, 1);
+$darken-bg-color: darken($primary-bg-color, 2%);
+$dark-shadow-primary-color: hsla(210, 35, 78, 1);
+$light-shadow-primary-color: hsla(0, 0, 100, 1);
+$dark-font-color: hsla(213, 51, 20, 1);
+$regular-hover-color: hsla(195, 98, 89, 1);
+$darken-regular-hover-color: hsla(195, 98, 79, 1);
+$error-color: hsla(7, 91, 65, 1);
+$darken-error-color: darken($error-color, 2%);
+$success-color: hsla(98, 89, 82, 1);
+$darken-success-color: darken($success-color, 2%);
+$genius-color: hsla(49, 100, 60, 1);
+$darken-genius-color: darken($genius-color, 2%);
+
+//
+$light-shadow-primary-transparent: transparentize($light-shadow-primary-color, 0.5);
+$dark-shadow-primary-transparent: transparentize($dark-shadow-primary-color, 0.5);
+
+// Gradients
+$primary-gradient: linear-gradient(130deg, $darken-bg-color 0%, $primary-bg-color 100%);
+$hover-btn-gradient: linear-gradient(130deg, $darken-regular-hover-color 0%, $regular-hover-color 100%);
+$error-gradient: linear-gradient(130deg, $darken-error-color 0%, $error-color 100%);
+$success-gradient: linear-gradient(130deg, $darken-success-color 0%, $success-color 100%);
+$genius-gradient: linear-gradient(130deg, $darken-genius-color 0%, $genius-color 100%);
+
+// // Mirror Gradients
+// $primary-gradient-mirror: linear-gradient(130deg, $primary-bg-color 0%, $darken-bg-color 100%);
+// $hover-btn-gradient-mirror: linear-gradient(130deg, $regular-hover-color 0%, $darken-regular-hover-color 100%);
+$error-gradient-mirror: linear-gradient(130deg, $error-color 0%, $darken-error-color 100%);
+// $success-gradient-mirror: linear-gradient(130deg, $success-color 0%, $darken-success-color 100%);
+// $genius-gradient-mirror: linear-gradient(130deg, $genius-color 0%, $darken-genius-color 100%);
+
```

Comment:
🤯you are the Gandalf the White of Sass!! Great great work, thank you for tackling this!

Reply:
Thanks!